lib/helpers/finalizeSpecificationObject.js

```diff
@@ -1,4 +1,5 @@
 const parser = require('swagger-parser');
+const { isEmpty } = require('lodash');
```

Comment:
if lodash only import that particular function into context. Rather use something like `just-is-empty`